src/main/java/dinkplugin/message/NotificationBody.java

```diff
@@ -26,6 +26,7 @@
     @NotNull
     NotificationType type;
     String playerName;
+    String playerUrl;
```

Comment:
I would consider making this `transient` so it doesn't get serialized into the webhook.

I'm not sure if it matters (not a Java expert), but this can be Null so we might want the `@Nullable` decorator here.

Reply:
Added the `@Nullable` decorator. I don't know how to make it `transient` any tips? `@Transient` decorator doesn't seem to work.

It shouldn't be a big problem since Discord can handle the `NULL` value.

Reply:
Would just be `transient String playerUrl;`

Reply:
Became irrelevant with IProdigy's suggestion